classpaths.gradle

```diff
@@ -109,17 +109,18 @@ dependencies {
     dhUtil 'ant:ant:1.6.5'
 
     dhDb 'joda-time:joda-time:2.10',
-           "org.codehaus.groovy:groovy:$GROOVY_VERSION",
-           "org.codehaus.groovy:groovy-json:$GROOVY_VERSION",
-           'com.jcraft:jsch:0.1.51',
-           "org.eclipse.jgit:org.eclipse.jgit:$Classpaths.JGIT_VERSION",
-           "org.eclipse.jgit:org.eclipse.jgit.ssh.jsch:$Classpaths.JGIT_VERSION",
-           'org.scala-lang:scala-library-all:2.12.2',
-           'org.scala-lang:scala-compiler:2.12.2',
-           'org.yaml:snakeyaml:1.22',
-           'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.8',
-           'com.github.javaparser:javaparser-core:2.0.0',
-           'com.tdunning:t-digest:3.2'
+            "org.codehaus.groovy:groovy:$GROOVY_VERSION",
```

Comment:
Was this change intentional?

Reply:
I intentionally corrected bad whitespace on the left.
The only material change is a new dependency:
```
            'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.8',
```